web interface demo/table1.js

```diff
@@ -0,0 +1,56 @@
+(function(){
+			const button33 = document.getElementsByClassName('button')[0];
+			const button37 = document.getElementsByClassName('button')[1];
+			const button39 = document.getElementsByClassName('button')[2];
+			const contentTable = document.getElementById('content');
+			
+			const stateOfClass33 = [1,0,0,1,1,0,1,0]; //заглушка, массив с состояниями компьютеров в ауд. 33
+			
+			const stateOfClass37 = [0,1,1,0,0,1]; //заглушка, массив с состояниями компьютеров в ауд. 37
+			
+			const stateOfClass39 = [1,0,1]; //заглушка, массив с состояниями компьютеров в ауд. 39

```

Comment:
@GoverLabs So, on my side of vision, i see that button creation and room creation does not depend from each other. And it's not dynamic. Is it possible to make an object with next field: button, state.
When we get a JSON object, parse it in object right away? And then, every single object are responsible for it's button, and all other stuff we add???